reports/final/report.tex

```diff
@@ -1238,24 +1215,24 @@ \subsection{FPGA Verification} % ETHAN
 % Talk about FPGA automated sims, testbenches CI ETC
 
 FPGA code testing was a continuous process taking place throughout the
-development cycle. SystemVerilog modules are tested and verified using so called
-``testbenches''. Testbenches are modules that wrap around the module you would
-like to test and modulate the input signals through a variety of test cases
+development cycle. SystemVerilog modules are tested and verified using
+``testbenches''. Testbenches are modules that wrap around the module to be
```

Comment:
there's inconsistency here between the tildes and single quotes. intentional?

Reply:
If you use double quotes it will not be in the correct direction